src/thor/multimodal.cc

```diff
@@ -287,6 +287,9 @@ bool MultiModalPathAlgorithm::ExpandForward(GraphReader& graphreader,
     // any costs along paths and roads. We only do this once
     // so if its from a transition we don't need to do it again
     if (mode_ == travel_mode_t::kPedestrian && !from_transition) {
+      // TODO: this needs to update second_of_week: careful, needs to wrap around the
+      // end of the week and we also need to update the day_ and dow_ mask in that case
+      // maybe make it a TimeInfo function for convenience
```

Comment:
this is not right yet. I left a TODO in the main description of the PR. we're not looking at the `TimeInfo.local_time` because that's already corrected for timezone. we want the time without timezone. eventually it needs a new method in TimeInfo IMO

Reply:
maybe use the `TimeInfo::forward` method here too